ui/src/data-products/vertical-geometry/vertical-geometry-table-item.tsx

```diff
@@ -97,6 +113,15 @@ export const VerticalGeometryTableItem: React.FC<VerticalGeometryTableItemProps>
             <td className={styles['data-product-table__column--number']}>
                 {roundToPrecision(verticalGeometry.end.station, Precision.measurementMeterDistance)}
             </td>
+            <td>
+                {verticalGeometry.verticalCoordinateSystem ??
+                    t('data-products.vertical-geometry.unknown')}
+            </td>
+            <td>
+                {overlapsAnother
+                    ? t('data-products.vertical-geometry.overlaps-another')
+                    : undefined}
+            </td>
```

Comment:
Uusi huomiocolumni, jossa toistaiseksi näytetään vaan tieto siitä onko joku kaltevuusjakso toisen kanssa limittäin